companies/nielsen.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "nielsen",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Nielsen Holdings Plc",
+    "runs": [
+        "VisualDNA"
+    ],
+    "address": "ATTN: Legal Dept.\nNielsen\nNielsen House\nOxford Business Park South\nJohn Smith Drive\nOxford OX4 2WB\nUK",
```

Comment:
Please don't include the ATTN line and the company name in the address ([1](https://github.com/datenanfragen/data#addresses)).